core/src/main/java/org/pentaho/metaverse/impl/FileSystemLineageCollector.java

```diff
@@ -42,6 +42,7 @@
 import java.util.zip.ZipEntry;
 import java.util.zip.ZipOutputStream;
 
+@Deprecated
```

Comment:
be sure to add the @deprecated javadoc with this. that way everyone will know what it is replaced with.

http://docs.oracle.com/javase/7/docs/technotes/guides/javadoc/deprecation/deprecation.html